backend/open_api_spec.yml

```diff
@@ -0,0 +1,790 @@
+        name:
+          description: The user's name
+          type: string
+        email:
+          description: The user's email
+          type: string
+
+    collection_uuid:
+      description: A unique identifier of a Collection.
+      type: string
+    curator_tag:
+      description: A curator-provided tag that is unique within a Collection.
+      type: string
+    dataset_uuid:
+      description: A unique identifier of a Dataset.
+      type: string
+    geneset_uuid:
+      description: A unique identifier of a gene set.
+      type: string
+    visibility:
+      description: >-
+        Determines the visibility of the collection, that will either make the collection visible on the public
+        sites or only viewable through obfuscated URLs.
+      type: string
+      enum: [PUBLIC, PRIVATE]
+    distribution:
+      description: >-
+        Cellxgene runs a heuristic to detect the approximate distribution of the data in X so that it can accurately
+        calculate statistical properties of the data. This field enables the curator to override this heuristic
+        and specify the data distribution explicitly.
+      type: string
+      enum: [COUNT, NORMAL]
+    is_primary_data:
+      description: >-
+        Describes whether cellular observations for this dataset are all canonical (PRIMARY),
+        all non-canonical (SECONDARY), or contain a mixture (BOTH).
+      type: string
+      enum: [PRIMARY, SECONDARY, BOTH]
+    links:
+      type: array
+      items:
+        type: object
+        additionalProperties: false
+        required:
+          - link_url
+          - link_type
+        properties:
+          link_name:
+            type: string
+          link_url:
+            type: string
+          link_type:
+            type: string
+            enum: [PROTOCOL, RAW_DATA, DOI, LAB_WEBSITE, OTHER, DATA_SOURCE]
+    dataset:
+      type: object
+      required:
+        - id
+        - curator_tag
+        - name
+        - revision
+        - collection_id
+        - collection_visibility
+        - revised_at
+        - is_primary_data
+        - X_normalization
+        - dataset_assets
+        - tombstone
+        - assay
+        - organism
+        - tissue
+      properties:
+        id:
+          $ref: "#/components/schemas/dataset_uuid"
+        curator_tag:
+          $ref: "#/components/schemas/curator_tag"
+        name:
+          type: string
+        revision:
+          type: integer
+        collection_id:
+          $ref: "#/components/schemas/collection_uuid"
+        collection_visibility:
+          $ref: "#/components/schemas/visibility"
+        revised_at:
+          type: number
+        is_primary_data:
+          $ref: "#/components/schemas/is_primary_data"
+        X_normalization:
+          type: string
+        dataset_assets:
+          type: array
+          items:
+            $ref: "#/components/schemas/dataset_asset"
+        tombstone:
```

Comment:
Should tombstone status be exposed to the API user?

Reply:
Yes. Jason wants to see deleted datasets as well.